### PR TITLE
(WIP) Theming of sidebar

### PR DESCRIPTION
**Pull request by MalcolmMielle**
- Add Welcome page and list page.
- Change in the theming of the sidebar to conform with libadwaita demo.
- Add combo row element.
- Changed popover to allow uniform card color in dark theme.

### CODE REVIEW

**Comment by prateekmedia** [comment]
@MalcolmMielle What's left implementing in this PR?

**Comment by MalcolmMielle** [comment]
Fixing the color/border of the popover

**Comment by prateekmedia** [comment]
I think we can make colors.dart named file which exports src/utils/colors.dart, So internal implementation warning would not be there.

**Comment by MalcolmMielle** [comment]
OK, this is the best I got for the time I have.

This PR changes are:
* Change in the theming of the sidebar to conform with libadwaita demo.
* Add combo row element.
* Add Welcome page and list page.
* Changed popover to allow uniform card color in dark theme.

File: lib/src/internal/popover.dart [hunk 1]
```diff
@@ -1,5 +1,50 @@
 import 'package:flutter/material.dart';
-import 'package:libadwaita/src/internal/triangle_painter.dart';
+import 'package:flutter/widgets.dart';
```
Comment (prateekmedia) [inline lib/src/internal/popover.dart:2]: This is a subset of material.dart so it's not needed.

**Comment by MalcolmMielle** [comment]
Screenshots:
Screenshot from 2021-12-13 18-28-27
Screenshot from 2021-12-13 18-28-37

Screenshot from 2021-12-13 18-28-15

**Comment by MalcolmMielle** [comment]
One of the limitation is that the popover top corners are squarred but I didn't have time to change that

**Comment by prateekmedia** [comment]
@MalcolmMielle Thank you for this! Also are you interested in joining the Telegram group I talked about earlier?

**Comment by MalcolmMielle** [comment]
You merged too quickly, hahaha. I solved your comment, move the import in libadwiata.dart, and removed a little bug in a new commit ;). Those were all good suggestions

**Comment by prateekmedia** [comment]
@MalcolmMielle I thought you were busy, Anyways you can directly commit those minor changes to this repository.

**Comment by MalcolmMielle** [comment]
Will do :)

**Pull request merged**